frontend/src/bundles/auth/auth-api.ts

```diff
@@ -33,7 +34,11 @@ class AuthApi extends HttpApiBase {
             },
         );
 
-        return await response.json<UserSignUpResponseDto>();
+        const data = await response.json<UserSignUpResponseDto>();
+
+        this.Storage.set(StorageKey.TOKEN, data.token);
+
+        return data;
```

Comment:
Looks like it should be in the store - frontend/src/bundles/auth/store/actions.ts

Reply:
Hi, I want to clarify this. You want me to move token storage logic to auth store actions?